OrbitCore/Pdb.h

```diff
@@ -33,10 +33,6 @@ class Pdb {
 
```

Comment:
As far as I can see there are no virtual functions anymore. So you could remove the virtual destructor. (Same for the linux class below.

Reply:
Done, thank you!